.github/actions/update-csv-bundles/action.yml

```diff
@@ -0,0 +1,173 @@
+name: "Update CSV bundles"
+description: "Generate and finalize OLM CSV bundles for a given version"
+
+inputs:
+  version:
+    description: "Version without v-prefix"
+    required: true
+  token:
+    description: "GitHub token to create PRs"
+    required: true
+  changelog-path:
+    description: "Path to the changelog file"
+    required: false
+
+outputs:
+  major_minor:
+    description: "Major.minor part of the version"
+    value: ${{ steps.vars.outputs.major_minor }}
+
+runs:
+  using: "composite"
+  steps:
+    - name: Derive release branch name
+      id: vars
+      shell: bash
+      run: |
+        VERSION="${{ inputs.version }}"
+
+        MAJOR_MINOR="${VERSION%.*}"
+        echo "major_minor=${MAJOR_MINOR}" >> "$GITHUB_OUTPUT"
+    - name: Checkout release branch
+      uses: actions/checkout@v4
```

Comment:
Could you please pin this dep by hash?

Reply:
https://github.com/Dynatrace/dynatrace-operator/pull/5794/commits/80835626c68670cc558effcb830347655c07de08

https://github.com/Dynatrace/dynatrace-operator/pull/5794/commits/3419bc67664235adf7582fa0211379e11938db2b